### DISCUSSION TITLE
OrangeFS multiple volumes and snapshots

### DISCUSSION THREAD

**Discussion by indivarnair** [waltligon/orangefs#98] [Q&A]
Hello All,

I am new to OrangeFS and have been going through the documentation.
I could not find anything on creating volumes or snapshots.

We would like to create a new volume for each project and take regular snapshots of the volume for deleted file recovery and backups. Is it possible on OrangeFS? 
If not, is there some workaround?

Would using LVM or ZFS help in doing this?

Snapshots are more important for us than creating multiple volumes.
Just having the ability to take regular snapshots will help too.

Thanks,

Regards,

Indivar Nair

**Comment by boydwilson** [comment]
Indivar,
With OrangeFS, to keep the performance optimized, we rely on lower-level
storage layers to handle things like snapshots.  The key is to make sure
you do a snapshot across all the devices/volumes at the same time during
lower activity.  Also make sure you cover the metadata volumes also.  Make
sure you are using LMDB, it is consistent and can be cp'd where the old BDB
is not.

-b

**Reply by indivarnair** [reply]
Thanks, Boyd,

So once we have created all the snapshots (for metadata, data on all servers) at the lower level, can we add them as a new filesystem in OrangeFS?

Regards,


Indivar Nair
